dev/bots/deploy_gallery.sh

```diff
@@ -35,21 +35,6 @@ if [[ "$OS" == "linux" ]]; then
     cd dev/integration_tests/flutter_gallery
     flutter build apk --release -t lib/main_publish.dart
   )
-  echo "Android Flutter Gallery built"
-  if [[ -z "$CIRRUS_PR" && "$CIRRUS_BRANCH" == "dev" && "$version" != *"pre"* ]]; then
-    echo "Deploying Flutter Gallery $version to Play Store..."
-    set +x # Don't echo back the below.
-    if [ -n "$ANDROID_GALLERY_UPLOAD_KEY" ]; then
-      echo "$ANDROID_GALLERY_UPLOAD_KEY" | base64 --decode > /root/.android/debug.keystore
-    fi
-    set -x
-    (
-      cd dev/integration_tests/flutter_gallery/android
-      bundle exec fastlane deploy_play_store
```

Comment:
you can remove all fastlane related code in the codebase as well. There are some in .cirrus.yaml. You can skip fastlane related setup in the dockerfile as well. Also the various fastlane related files in flutter_gallery/android|ios/fastlane.

Reply:
Please help take another look.
`Gemfile.lock` was updated after I ran `bundle` after removing `fastlane` codes from `Gemfile`.